contrib/ckan/docker-entrypoint.d/openafrica_ckan.sh

```diff
@@ -29,4 +36,8 @@ ckan config-tool ${CKAN_INI} -s app:main -e ckan.preview.text_formats='text plai
 # Customize which image formats the image_view plugin will show
 ckan config-tool ${CKAN_INI} -s app:main -e ckan.preview.image_formats='png jpeg jpg gif'
 
+# reCAPTCHA
+ckan config-tool ${CKAN_INI} -s app:main -e ckan.recaptcha.publickey=${CKAN_RECAPTCHA_PUBLIC_KEY}
+ckan config-tool ${CKAN_INI} -s app:main -e ckan.recaptcha.privatekey=${CKAN_RECAPTCHA_PRIVATE_KEY}
```

Comment:
Are these recaptcha keys the same one in **_openAFRICA | Dokku_** item in the vault?

Reply:
Yes these are the ones added to Vault